charts/bulk-scan-orchestrator/values.template.yaml

```diff
@@ -0,0 +1,37 @@
+java:
+  applicationPort: 8582
+  readinessPath: '/health'
+  livenessPath: '/health'
```

Comment:
why are these values set?
this line and the above

readiness is at the default and we don't want people overriding that for no reason, and adding a liveness check is adding a 1 line spring-boot starter